exonum-java-binding-core/rust/src/runtime/cmd.rs

```diff
@@ -1,21 +1,25 @@
-use super::{Config, JvmConfig, ServiceConfig};
+use super::{Config, JvmConfig, EjbConfig, ServiceConfig};
 use exonum::helpers::fabric::keys;
 use exonum::helpers::fabric::Argument;
 use exonum::helpers::fabric::CommandExtension;
 use exonum::helpers::fabric::Context;
 use exonum::node::NodeConfig;
 use failure;
+use std::collections::BTreeMap;
 use toml::Value;
 
-const EJB_JVM_ARGUMENTS: &str = "EJB_JVM_ARGUMENTS";
-const EJB_JVM_DEBUG_SOCKET: &str = "EJB_JVM_DEBUG_SOCKET";
 const EJB_LOG_CONFIG_PATH: &str = "EJB_LOG_CONFIG_PATH";
 const EJB_CLASSPATH: &str = "EJB_CLASSPATH";
 const EJB_LIBPATH: &str = "EJB_LIBPATH";
 const EJB_MODULE_NAME: &str = "EJB_MODULE_NAME";
 const EJB_PORT: &str = "EJB_PORT";
+const JVM_DEBUG_SOCKET: &str = "JVM_DEBUG_SOCKET";
+const JVM_ARGS_PREPEND: &str = "JVM_ARGS_PREPEND";
+const JVM_ARGS_APPEND: &str = "JVM_ARGS_APPEND";
 const EJB_JVM_CONFIG_NAME: &str = "ejb_jvm_config";
 pub const EJB_CONFIG_NAME: &str = "ejb";
+const EJB_CONFIG_KEY: &str = "ejb_config";
```

Comment:
What do `_NAME` and `_KEY` suffixes mean? They seem to be used rather randomly. Will namespacing them help?